src/Glpi/Application/View/TemplateRenderer.php

```diff
@@ -118,6 +120,15 @@ public function __construct(string $rootdir = GLPI_ROOT, ?string $cachedir = nul
         $this->environment->addExtension(new SearchExtension());
         $this->environment->addExtension(new SessionExtension());
         $this->environment->addExtension(new TeamExtension());
+        if (
+            in_array(GLPI_ENVIRONMENT_TYPE, [\Glpi\Application\Environment::DEVELOPMENT->value, \Glpi\Application\Environment::TESTING->value])
+            && defined('GLPI_STRICT_ENV')
+            && GLPI_STRICT_ENV
```

Comment:
From what I understand, `GLPI_STRICT_ENV` is only set to true on testing.
![image](https://github.com/user-attachments/assets/bcc50617-c03b-42f7-84ab-4a1d5f2bc3ea)

So you are checking for a dev or testing env but the condition can only be true for the testing env unless some custom const were injected in your local_define.php file.

Is is on purpose ?

Also, since `GLPI_STRICT_ENV` is already conditioned by the current environment, maybe you could just check this instead ?
```suggestion
            defined('GLPI_STRICT_ENV') && GLPI_STRICT_ENV
```

Reply:
What I want strict warning can be enabled in both testing and development environment.

But for development, the initial idea (when I setted that up weeks ago) was to not enforce it. But maybe it should be always enabled.

Reply:
> But maybe it should be always enabled.

Yes, it should.

You could test for `\Glpi\Application\Environment::get()->shouldEnableExtraDevAndDebugTools() || GLPI_STRICT_ENV`.